docs/src/main/asciidoc/_developer-guide/Migrate_Seam_2_Booking_EAR_Binaries_-_Step_by_Step.adoc

```diff
@@ -123,9 +123,9 @@ this, we create a `jboss-deployment-structure.xml` file in the EAR
 ----
 
 In the "deployment" section, we add the dependency for the
-`javax.faces.api` for the JSF 1.2 module. We also add the dependency for
-the JSF 1.2 module in the sub-deployment section for the WAR and exclude
-the module for JSF 2.0.
+`javax.faces.api` for the Faces 1.2 module. We also add the dependency for
+the Faces 1.2 module in the sub-deployment section for the WAR and exclude
+the module for Faces 2.0.
```

Comment:
Please see above comment regarding JSF/FACES 1.2/2.0 which I think will be replaced with `Jakarta Server Faces 2.3`.